engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -117,8 +115,10 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   }
 
   when(FetchingCachedOutputsFromDatabase) {
-    case Event(CachedOutputLookupSucceeded(wdlValueSimpletons, jobDetritus, returnCode, cacheHit), data: ResponsePendingData) =>
-      makeBackendCopyCacheHit(cacheHit, wdlValueSimpletons, jobDetritus, returnCode, data)
+    case Event(CachedOutputLookupSucceeded(wdlValueSimpletons, jobDetritus, returnCode, cacheResultId, cacheHitDetails), data: ResponsePendingData) =>
+      writeToMetadata(Map(callCachingReadResultMetadataKey -> s"Cache Hit: $cacheHitDetails"))
+      log.debug("Cache hit for {}! Fetching cached result {}", jobTag, cacheResultId)
```

Comment:
ToL: Are there any other `log.info()` messages to let the user know that call caching has ~actived~ hit? Otherwise, with a long running copy, I feel like we're super verbose about running, then _nothing_ when we cache hit.